doc/dependencies.md

```diff
@@ -11,7 +11,7 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | Expat | [2.2.7](https://libexpat.github.io/) |  | No | Yes |  |
 | fontconfig | [2.12.1](https://www.freedesktop.org/software/fontconfig/release/) |  | No | Yes |  |
 | FreeType | [2.7.1](https://download.savannah.gnu.org/releases/freetype) |  | No |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) (Android only) |
-| GCC |  | [4.8+](https://gcc.gnu.org/) (C++11 support) |  |  |  |
+| GCC |  | [5.1+](https://gcc.gnu.org/) (C++11 support) |  |  |  |
```

Comment:
```suggestion
| GCC |  | [4.8+ for Linux or 5.1+ for Windows cross compilation](https://gcc.gnu.org/) (C++11 support) |  |  |  |
```

4.8 works just fine on Linux. And one of our ci runs is using 4.8.

Reply:
I'd rather not. We don't actually need to maintain compile-time compatibility with CentOS 7 (for some reason it has became a big focus recently), we really only need runtime back-compat. I'm currently writing up some documentation to clarify our build and runtime requirements, and what we actually do and don't need to support. This is also going to change when we require C++17, so I don't see the point in making a distinction at this stage.

Reply:
>  We don't actually need to maintain compile-time compatibility with CentOS 7 

I also think this is pointless FWIW. We need to move forward, there is no need to try to support increasingly old platforms. That's just make-work. And it's moot with C++17 anyway.

Reply:
CentOS 8 was released already, and I haven't heard any discussion about supporting anything older than the most recent stable release of major distros?

Reply:
I am not aware of this policy. If we stop supporting second-to-last LTS, we will also drop support for Ubuntu Bionic, which is the primary Ubuntu platform that Bitcoin Core runs on.

I think dropping of OSes should only be done on a case-by-case basis with all tradeoffs in mind.